Assets/Scripts/ECS/Events/DeathCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Unity.Entities;
+
+namespace Ecosystem.ECS.Events
+{
+    /// <summary>
+    /// Marks the entity to command other entities to die
+    /// </summary>
+    public struct DeathCommand : IComponentData
+    {
+        public Entity target;
```

Comment:
Public fields start with capital letters
```suggestion
        public Entity Target;
```